### DISCUSSION TITLE
GitKraken Pro For Students

### DISCUSSION THREAD

**Discussion by wcordova** [community/community#142184] [GitHub Education]
### Hi everyone,

I have consulted gitkraken and they tell me that the teachers are included in the development pack, but I don't have access as a pro.

**Comment by davevad93** [comment]
Hi @wcordova ,

Try going to the [pack offers page](https://education.github.com/pack), scroll down to the "***All offers***" section and try to redeem your [Free GitKraken Pro license](https://education.github.com/pack/redeem/gitkraken-student) there, if you can't redeem it, I suggest you to raise a ticket [here](https://support.github.com/contact/education), select "`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue.
